src/components/PostInfo/PostInfo.tsx

```diff
@@ -1,5 +1,51 @@
 import React from 'react';
+import { Post } from '../../types/types';
+import { CommentList } from '../CommentList';
+import usersFromServer from '../../api/users';
+import commentsFromServer from '../../api/comments';
+import { UserInfo } from '../UserInfo';
 
-export const PostInfo: React.FC = () => (
-  <>Put the post here</>
-);
+function filterCommets(comments: Comment[], postOfId: number) {
+  return [...comments].filter(
+    comment => comment.postId === postOfId,
+  );
+}
+
+export const PostInfo: React.FC<Post> = ({
+  title,
+  body,
+  userId,
+  id,
+}) => {
+  const filteredCommets = filterCommets(commentsFromServer, id);
```

Comment:
It's recommended to fix the variable name `filteredCommets` to `filteredComments` for better readability and consistency with the function name change.